docs/guides/player-workflows.md

```diff
@@ -0,0 +1,58 @@
+# Player Workflows
+
```

Comment:
Yeah, this was the "removing players" document. I think it makes sense to have a document devoted to "doing things with players other than creating them". That's the intent of this.